trunk/SUAVE/Components/Energy/Storages/Batteries/Constant_Mass/Lithium_Ion_LiFePO4_18650.py

```diff
@@ -0,0 +1,362 @@
+        Outputs:
+        None
+        
+        Properties Used:
+        N/A
+        """ 
+    def __defaults__(self):
+        self.tag                              = 'Lithium_Ion_LiFePO4_Cell' 
+         
+        self.cell.diameter                    = 0.0185                                                   # [m]
+        self.cell.height                      = 0.0653                                                   # [m]
+        self.cell.mass                        = 0.03  * Units.kg                                         # [kg]
+        self.cell.surface_area                = (np.pi*self.cell.height*self.cell.diameter) + (0.5*np.pi*self.cell.diameter**2)  # [m^2]
+        self.cell.volume                      = np.pi*(0.5*self.cell.diameter)**2*self.cell.height       # [m^3] 
+        self.cell.density                     = self.cell.mass/self.cell.volume                          # [kg/m^3]
+        self.cell.electrode_area              = 0.0342                                                   # [m^2]  # estimated 
+                                                        
+        self.cell.max_voltage                 = 3.6                                                      # [V]
+        self.cell.nominal_capacity            = 1.5                                                      # [Amp-Hrs]
+        self.cell.nominal_voltage             = 3.6                                                      # [V]
+        self.cell.charging_voltage            = self.cell.nominal_voltage                                # [V]  
+         
+        self.watt_hour_rating                 = self.cell.nominal_capacity  * self.cell.nominal_voltage  # [Watt-hours]      
+        self.specific_energy                  = self.watt_hour_rating*Units.Wh/self.cell.mass            # [J/kg]
+        self.specific_power                   = self.specific_energy/self.cell.nominal_capacity          # [W/kg]   
+        self.ragone.const_1                   = 88.818  * Units.kW/Units.kg
+        self.ragone.const_2                   = -.01533 / (Units.Wh/Units.kg)
+        self.ragone.lower_bound               = 60.     * Units.Wh/Units.kg
+        self.ragone.upper_bound               = 225.    * Units.Wh/Units.kg         
+        self.resistance                       = 0.022                                                    # [Ohms]
+                                                        
+        self.specific_heat_capacity           = 1115                                                     # [J/kgK] 
+        self.cell.specific_heat_capacity      = 1115                                                     # [J/kgK] 
+        self.cell.radial_thermal_conductivity = 0.475                                                    # [J/kgK]  
+        self.cell.axial_thermal_conductivity  = 37.6                                                     # [J/kgK]   
+        
+        return   
+
+    def energy_calc(self,numerics,battery_discharge_flag= True): 
```

Comment:
It looks like this is the same function as the base energy_calc in the Lithium_Ion general class; since it is inheriting Lithium_Ion I don't think you need to repeat this function here for the LFP

Reply:
Repeated functions removed.